backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -19,15 +19,18 @@ package object backend {
     private val indexString = index map { _.toString } getOrElse "NA"
     val tag = s"${call.fullyQualifiedName}:$indexString:$attempt"
     val isShard = index.isDefined
+    def toString(workflowId: WorkflowId) = s"$workflowId:$this"
```

Comment:
can this be called something other than `toString` as there's an expectation that `toString` is a `Function0[String]`